<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DebugInfos</class>
 <widget class="QDialog" name="DebugInfos">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Information</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetMaximumSize</enum>
       </property>
       <item>
        <widget class="QLabel" name="PlaylistTitle">
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="UserAgent">
         <property name="autoFillBackground">
          <bool>false</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="text">
          <string>-</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="line">
         <property name="styleSheet">
          <string notr="true">background-color: transparent;</string>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="FreeSystemMemory">
           <property name="styleSheet">
            <string notr="true">background-color: transparent;</string>
           </property>
           <property name="text">
            <string>Free System Memory:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="TotalSystemMemory">
           <property name="styleSheet">
            <string notr="true">background-color: transparent;</string>
           </property>
           <property name="text">
            <string>Total System Memory:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="OutputMemoryUse">
           <property name="styleSheet">
            <string notr="true">background-color: transparent;</string>
           </property>
           <property name="text">
            <string>Memory App use:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="MaxMemoryUsed">
           <property name="styleSheet">
            <string notr="true">background-color: transparent;</string>
           </property>
           <property name="text">
            <string>Max Memory App used:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="TotalSpace">
           <property name="text">
            <string>Free Space:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="FreeSpace">
           <property name="text">
            <string>Total Space</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="ThreadsNumber">
           <property name="styleSheet">
            <string notr="true">background-color: transparent;</string>
           </property>
           <property name="text">
            <string>Threads:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="MaxThreadsNumber">
           <property name="styleSheet">
            <string notr="true">background-color: transparent;</string>
           </property>
           <property name="text">
            <string>Max Threads:</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QPushButton" name="open_config">
           <property name="text">
            <string>open config dialog</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="close_player">
           <property name="text">
            <string>close player</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="close_dialog">
           <property name="text">
            <string>close dialog</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
